test/lint/lint-python.py

```diff
@@ -5,74 +5,25 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 """
-Check for specified flake8 and mypy warnings in python files.
+Check for specified mypy warnings in python files.
 """
 
 import os
 from pathlib import Path
 import subprocess
-import sys
 
 from importlib.metadata import metadata, PackageNotFoundError
 
 # Customize mypy cache dir via environment variable
 cache_dir = Path(__file__).parent.parent / ".mypy_cache"
 os.environ["MYPY_CACHE_DIR"] = str(cache_dir)
 
-DEPS = ['flake8', 'lief', 'mypy', 'pyzmq']
-
-# All .py files, except those in src/ (to exclude subtrees there)
-FLAKE_FILES_ARGS = ['git', 'ls-files', '*.py', ':!:src/*.py']
+DEPS = ['lief', 'mypy', 'pyzmq']
 
 # Only .py files in test/functional and contrib/devtools have type annotations
 # enforced.
 MYPY_FILES_ARGS = ['git', 'ls-files', 'test/functional/*.py', 'contrib/devtools/*.py']
 
-ENABLED = (
-    'E101,'  # indentation contains mixed spaces and tabs
-    'E401,'  # multiple imports on one line
-    'E402,'  # module level import not at top of file
-    'E701,'  # multiple statements on one line (colon)
-    'E702,'  # multiple statements on one line (semicolon)
-    'E703,'  # statement ends with a semicolon
-    'E711,'  # comparison to None should be 'if cond is None:'
-    'E714,'  # test for object identity should be "is not"
-    'E721,'  # do not compare types, use "isinstance()"
-    'E722,'  # do not use bare 'except'
-    'E742,'  # do not define classes named "l", "O", or "I"
-    'E743,'  # do not define functions named "l", "O", or "I"
-    'E901,'  # SyntaxError: invalid syntax
```

Comment:
nit: this should've been removed in the [earlier commit](https://github.com/bitcoin/bitcoin/pull/30723/commits/66668f5292e0988461e5e34654c2b00dbbcb798e) that dropped syntaxerror checks